### DISCUSSION TITLE
OG plots farming BUG

### DISCUSSION THREAD

**Discussion by tracy32792** [Chia-Network/chia-blockchain#8073] [Support]
Bug's wallet appears to be functioning normally. There is synchronization and there is preliminary screening. But the farm inside the Full Node Connection Kib upload is always 0. Download data is also available. Also asked around a lot of people have the same mistake. I hope it's official

**Comment by tracy32792** [comment]
**my farm inside the Full Node Connection Kib upload and download both are zero** i need help,

**Comment by BrandtH22** [comment]
@tracy32792 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
